tests/integration/templates/template37.yaml

```diff
@@ -0,0 +1,78 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  Environment:
+    Type: String
+    Default: 'companyname-ci'
+
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      CidrBlock: "100.0.0.0/20"
+
+  RouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId:
+        Ref: VPC
+      Tags:
+      - Key: env
+        Value: production
+
+  SubnetA:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone:
+        Fn::Select:
+        - 0
+        - Fn::GetAZs:
+            Ref: AWS::Region
+      CidrBlock: "100.0.0.0/24"
+      VpcId:
+        Ref: VPC
+
+  SubnetB:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone:
+        Fn::Select:
+        - 0
+        - Fn::GetAZs:
+            Ref: AWS::Region
+      CidrBlock: "100.0.2.0/24"
+      VpcId:
+        Ref: VPC
+
+  RouteTableAssociationA:
+    Type: "AWS::EC2::SubnetRouteTableAssociation"
+    Properties:
+      SubnetId:
+        Ref: SubnetA
+      RouteTableId:
+        Ref: RouteTable
+      Main: false
+    DependsOn:
+      - SubnetA
+      - RouteTable
+
+  RouteTableAssociationB:
+    Type: "AWS::EC2::SubnetRouteTableAssociation"
+    Properties:
+      SubnetId:
+        Ref: SubnetB
+      RouteTableId:
+        Ref: RouteTable
+      Main: false
+    DependsOn:
+      - SubnetB
+      - RouteTable
+
+
+Outputs:
+  PublicRoute:
+    Value:
+      Ref: PublicRoute
+    Export:
+      Name: 'publicRoute-identify'
```

Comment:
```suggestion
```

Also seems unused and unresolvable

Reply:
I change the output list to have the route table id as output for test.